staubli_tx2_60_support/urdf/tx2_60_macro.xacro

```diff
@@ -0,0 +1,239 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://wiki.ros.org/xacro">
+  <xacro:include filename="$(find staubli_resources)/urdf/common_materials.xacro"/>
+
+  <xacro:macro name="staubli_tx2_60" params="prefix">
+    <!-- links: main serial chain -->
+    <link name="${prefix}base_link">
+      <visual>
+        <origin xyz="0 0 0" rpy="0 0 0"/>
+        <geometry>
+          <mesh filename="package://staubli_tx2_60_support/meshes/tx2_60/visual/base_link.stl"/>
+        </geometry>
+        <xacro:material_staubli_ral_melon_yellow />
+      </visual>
+      <collision>
+        <origin xyz="0 0 0" rpy="0 0 0"/>
+        <geometry>
+          <mesh filename="package://staubli_tx2_60_support/meshes/tx2_60/collision/base_link.stl"/>
+        </geometry>
+      </collision>
+      <inertial>
+          <mass value="4.65971957"/>
+          <origin xyz="-0.00257355 -0.00039402 0.08786146" rpy="0.0 0.0 0.0"/>
+          <inertia ixx="0.02319781" ixy="0.00023234" ixz="0.00162421"
+                                    iyy="0.02621322" iyz="0.00008092"
+                                                     izz="0.02286303"/>
+      </inertial>
```

Comment:
Could you clarify where you got these values from (not necessarily these values specifically, but all `inertial` values)? Are they copied from some other model?

Reply:
The inertia values are copied from the TX60 package.
It is written in the package.xml
```
Masses, center of mass and moments of inertia were copied 
      from the TX60 package. There they were calculated using Solidworks and the
      official Staubli CAD models, and may not be accurate.
```

Reply:
Hm. I'm inclined to say the same things as in #41: let's remove these elements then as copying them doesn't really make sense.

If you really want to keep them, please remove this part:

> There they were calculated using Solidworks and the official Staubli CAD models, and may not be accurate.

as it's confusing: were they copied or calculated using SolidWorks? Perhaps something like this would be clearer:

> See the manifest of `staubli_tx60_support` for more information on how these values were derived.

Reply:
I am fine with removing them